### PR TITLE
rendering v0.1 as a distinct snapshot?

### PR DESCRIPTION
**Pull request by bumblefudge**
I opened a separate PR because this might be superfluous given the "versions" script I found but couldn't understand, but I looked at [the way distinct versions are pulled out in the Pres Ex spec](https://github.com/decentralized-identity/presentation-exchange/blob/master/specs.json) and tried to replicate it here in case it would help with #131 ? (I preserved the web.archive version as well just in case)

### CODE REVIEW

**Review by OR13 — COMMENTED** [review]
@csuwildcat is this the recommended way to do this?

**Review by OR13 — CHANGES_REQUESTED** [review]
@bumblefudge please refer to the spec up documentation to justify this approach, I will approve when the spec up readme says what you have done here is recommended.

File: spec/specs.json [hunk 1]
```diff
@@ -12,6 +12,18 @@
         "account": "decentralized-identity",
         "repo": "waci-presentation-exchange"
       }
-    }
+    },
+    {
+      "title": "Wallet And Credential Interactions",
+      "spec_directory": "./v0.1",
+      "output_path": "./build/v0.1",
```
Comment (csuwildcat) [inline spec/specs.json:19]: Might want to do v0.1.0

**Review by csuwildcat — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by brentzundel** [comment]
multiple positive reviews.

**Pull request merged**